mmeval/metrics/precision_recall_f1score.py

```diff
@@ -194,10 +251,10 @@ class SingleLabelMetric(BaseMetric):
     """
 
     def __init__(self,
-                 thrs: Union[float, Sequence[Optional[float]], None] = 0.,
+                 num_classes: Optional[int] = None,
+                 thrs: Union[float, Sequence[Optional[float]], None] = None,
                  items: Sequence[str] = ('precision', 'recall', 'f1-score'),
                  average: Optional[str] = 'macro',
-                 num_classes: Optional[int] = None,
                  **kwargs) -> None:
         super().__init__(**kwargs)
```

Comment:
Arguments order is changed. This might be BC-breaking for users. Is this change necessary?

Reply:
In previous version, `SIngleLabelMetric` and `MultiLabelMetric` are seperated.

After split `MultiLabelMetric` into two pieces, `MultiLabelPrecisionRecallF1score` and `SingleLabelPrecisionRecallF1score` shares the same interface, since `num_classes` is necessary for `MultiLabelPrecisionRecallF1score`, change here to make consistency arguments for both metrics.